### ISSUE TITLE
Renaming branches, making releases

### ISSUE THREAD

**Issue by bobturneruk** [AFM-SPM/TopoStats#230]
**Is your feature request related to a problem? Please describe.**
It is currently difficult to find what version of topostats to use. In the current system we have a `master` and a `dev` branch. The `master` branch is the "default" branch and hasn't been altered for months. You can see the `dev` branch and associated branches here:
![image](https://user-images.githubusercontent.com/34244196/180185699-9cfc10b8-fe80-4f86-918f-b522841b5c8c.png)
Every time an issue is addressed, feature is added or bug is fixed, a "pull request" is made and code is merged into the `dev` branch.

The first branch that users see, however, is `master` as this is set to be the default branch.

**Describe the solution you'd like**
This has been discussed. `master` should be renamed `legacy` and `dev` should be renamed `main` and made default so it comes up right away when this URL is used https://github.com/AFM-SPM/TopoStats

However, this did provoke some questions about how we do subsequent development. I propose that we no longer have a `dev` branch and future pull requests are made against `main`. At this point we should begin "releasing" versions of topostats for general use.

From a development perspective the release process is:
- We pick a commit on `main` that we want to release from.
- We "tag" this with a semantic version number e.g. 2.0.0 e.g. via this interface https://github.com/AFM-SPM/TopoStats/releases/new.
- This creates a zip file / tar.gz file with a snapshot of topostats that we call version 2.0.0 (in this example).
- We add this to PyPI.

Once this is done, a user installs the latest release by:
- Install Python
- Optionally but recommended, make a clean virtual environment
- `pip install topostats`

**Describe alternatives you've considered**
This is a pretty standard way of doing things in Python see e.g. https://github.com/scikit-image/scikit-image

**Comment by ns-rse** [comment]
Closing as we [released last week](https://github.com/AFM-SPM/TopoStats/releases/tag/v2.0.0) :tada:
